dev/NavigationView/NavigationView.cpp

```diff
@@ -4040,18 +4059,34 @@ void NavigationView::OnIsPaneOpenChanged()
 
     if (SharedHelpers::IsThemeShadowAvailable())
     {
-        if (auto&& splitView = m_rootSplitView.get())
+        // Drop Shadows were only introduced in OS versions 21h1 or higher. Projected Shadows will be used for older versions.
+        if (SharedHelpers::Is21H1OrHigher())
```

Comment:
>Is21H1OrHigher [](start = 27, length = 14)

We don't actually care if theme shadow is available  to do drop shadows right? so slightly better would be:

```
if(SharedHelpers::Is21H1OrHigher())
{
...
}
else if (SharedHelpers::IsThemeShadowAvailable)
{
...
}

Reply:
I'm pretty sure Drop shadows still uses themeShadows - with `shadowCaster_uiElement10.Shadow(winrt::ThemeShadow{});`